core/src/PayPal/ShippingTracking/Builder/Node/TrackingItemsNodeBuilder.php

```diff
@@ -98,16 +98,22 @@ public function build(): array
             $name = $this->validateName($product, $productData);
 
             $item = [
-                'sku' => $sku,
-                'quantity' => $quantity,
                 'name' => $name,
-                'description' => $this->getProductDescription($productData),
+                'quantity' => $quantity,
+                'sku' => $sku,
+//                'description' => $this->getProductDescription($productData),
                 'url' => $this->getProductUrl($productData['product'], $productAttributeId),
                 'image_url' => $this->getProductImageUrl($productData['product'], $productAttributeId),
             ];
 
+            $allowedUpcTypes = ['UPC-A', 'UPC-B', 'UPC-C', 'UPC-D', 'UPC-E', 'UPC-2', 'UPC-5'];
```

Comment:
should we do this check in module (where it's hardcoded and can only be changed with a module upgrade) or on API side?